configs/_base_/datasets/semantickitti.py

```diff
@@ -2,13 +2,11 @@
 dataset_type = 'SemanticKITTIDataset'
```

Comment:
Should we rename this file to `semantickitti-seg.py`?

Reply:
I think semantickitti already indicates that is the semantic version of the kitti dataset.

Reply:
but we should repalce SemanticKITTIDataset with SemanticKittiDataset